common/pigx-runner.in

```diff
@@ -214,14 +214,15 @@ settings.
         defaults = path.join(dirs['locations']['pkgdatadir'], 'settings.yaml')
 
     if not path.exists(defaults):
-        bail("Could not find default settings.  Did you forget to set PIGX_UNINSTALLED?")
+        bail(f"Could not find default settings at {defaults}. Did you forget "
+                "to set PIGX_UNINSTALLED?")
```

Comment:
What's with the indentation after the line break?  It looks like an arbitrary number of spaces.

Reply:
That was the first tab over to ensure the bottom text starts on a position to the right of the start of the top text... Ill put the top text also on its own line so it doesnt look as awkward...

Reply:
Updated now